docs/logs/archiving.md

```diff
@@ -0,0 +1,146 @@
+title: Log Archiving
+description: Archiving logs to an S3-compatible object store
+
+<div style="padding-top:40px;display:grid;grid-template-columns:150px 150px 150px 150px;grid-template-rows:100px 100px;">
+  <a href="https://aws.amazon.com/s3" target="_blank" title="Amazon AWS S3">
+    <img src="../../images/logs/archiving/aws-logo.png" style="height:80px;width:134px;margin-top:auto;">
+  </a>
+  <a href="https://www.digitalocean.com/products/spaces" target="_blank" title="Digital Ocean Spaces">
+    <img src="../../images/logs/archiving/digital-ocean-logo.png" style="height:80px;width:80px;margin-top:auto;">
+  </a>
+  <a href="https://www.ibm.com/cloud/object-storage" target="_blank" title="IBM Cloud Object Storage">
+    <img src="../../images/logs/archiving/ibm-logo.png" style="height:80px;width:90px;margin-top:auto;">
+  </a>
+  <a href="https://azure.microsoft.com/services/storage" target="_blank" title="Azure Cloud Storage">
+    <img src="../../images/logs/archiving/azure-logo.png" style="height:80px;width:104px;margin-top:auto;">
+  </a>
+  <a href="https://cloud.google.com/storage" target="_blank" title="Google Cloud Storage" style="grid-column-start:2">
+    <img src="../../images/logs/archiving/google-logo.png" style="height:80px;width:99px;margin-top:auto;">
+  </a>
+  <a href="https://min.io" target="_blank" title="MinIO Object Storage" style="align-self:center;">
+    <img src="../../images/logs/archiving/minio-logo.png" style="height:30px;width:150px;margin-top:auto;">
+  </a>
+</div>
+
+You can configure a Logs App to forward all the logs it receives
+to an S3-compatible object store.
+
+### How do I configure archiving for my Logs App?
+
+In Sematext web app, go to `Integrations > Apps` view
+and choose `Configure S3` using row context menu (*three-dots* icon)
+of the app whose logs you want to ship to S3:
+
+For Amazon S3, all you have to provide are
+[credentials](#how-to-obtain-credentials-from-aws) and a bucket name:
+
+<img src="../../images/logs/archiving/aws-s3.png" style="height:400px;width:490px">
+
+For any other S3-compatible object store you'll also have to provide
+a service endpoint. Cloud object store providers, like *DigitalOcean
+Spaces* or *IBM Cloud Object Storage* usually refer to it as a 
+`Public Service Endpoint` in bucket configuration. For Minio users
+this would be your Minio public URL.
+
+<img src="../../images/logs/archiving/non-aws-s3.png" style="width:490px;height:477px;">
+
+Paste `Access key ID` and `Secret access key` to the corresponding
+fields.
+
+Enter `Bucket name` (just the simple name, not fully qualified ARN) and
+choose `Compression` (read on for more details about compression) and
+confirm with `Verify and save`.
+
+At this point, Logs App is going to check whether the information is
```

Comment:
At this point, Logs App is going to check whether the information is

=>

At this point, your Logs App is going to check if the information is